features/admin.actions.v1/components/user-attributes/user-attribute-list.tsx

```diff
@@ -186,6 +213,37 @@ const UserAttributeList: FunctionComponent<UserAttributeListPropsInterface> = ({
         );
     };
 
+    /**
+     * Limits the number of attributes per action.
+     *
+     * @param attributesList - List of attributes to be checked.
+     * @returns - whether the limit is reached.
+     */
+    const isMaxAttributesConfigured = (attributesList: Claim[]): boolean => {
+
+        if (attributesList?.length === ActionsConstants.MAX_ALLOWED_ATTRIBUTES_PRE_UPDATE_PROFILE) {
+            setIsAttributeLimitReached(true);
+
+            return true;
+        }
+
+        setIsAttributeLimitReached(false);
+
+        return false;
+    };
+
+    /**
+     * Disables the role claim attribute.
+     *
+     * The role attribute is temporarily disabled until the ambuiguities
+     * related to the claim are resolved.
+     * @param claimsList - List of claims.
+     * @returns - Filtered claims list.
+     */
+    const disableRoleClaimAttribute = (claimsList: Claim[]): Claim[] => {
```

Comment:
```suggestion
    const removeRoleClaimAttribute = (claimsList: Claim[]): Claim[] => {
```

disable can be confusing when someone is referring to this later.

Reply:
This is updated with 74bbdaf5a32146e39a7af466bb2dbd9aee5c2735